packages/mesos/build

```diff
@@ -17,8 +17,7 @@ LIBS="-lssl -lcrypto" LDFLAGS="-L/opt/mesosphere/active/openssl/lib" "/pkg/src/m
   --with-libevent=/opt/mesosphere/active/libevent \
   --with-curl=/opt/mesosphere/active/curl \
   --sbindir="$PKG_PATH/bin"
-# TODO(cmaloney): -j8 is almost certainly wrong.
-make -j8
```

Comment:
It looks like if you don't specify a number the amount of jobs is unbounded https://www.gnu.org/software/make/manual/html_node/Parallel.html

"If the ‘-j’ option is followed by an integer, this is the number of recipes to execute at once; this is called the number of job slots. If there is nothing looking like an integer after the ‘-j’ option, there is no limit on the number of job slots. The default number of job slots is one, which means serial execution (one thing at a time)."